src/views/referee.njk

```diff
@@ -0,0 +1,126 @@
+{% extends "_base.njk" %}
+
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% block content %}
+
+  {% set emailErrors %}
+    {% if model.refereeEmailEmptyError %}
+        <p>You must enter the email that can be used to contact your referee</p>
+    {% else %}
+        <p>You must enter a valid email address</p>
```

Comment:
These should either be
- `<p class="govuk-body">Lorem ipsum</p>` or 
- just plain `Lorem ipsum` strings and used further down as 
  ```
  errorMessage: {
    text: emailErrors
  } if model.refereeEmailEmptyError or model.refereeEmailInvalidError,
  ```

I'd prefer the plain string version as it'd be consistent with the other field's error message.

Reply:
Resolved in 835c3f6e21d690211769853ecc625e19d00292c6.